docs/introduction/basic_usage.rst

```diff
@@ -0,0 +1,20 @@
+###########
+Basic usage
+###########
+
+[describe the steps a user needs to do to get the system into a usable state for someone
```

Comment:
@intellectronica Oh I see, Paulo has (correctly) grabbed the example content from the application documentation starter pack, and that's what we're going to need to replace with the real thing.